### DISCUSSION TITLE
Handling of special characters

### DISCUSSION THREAD

**Discussion by EnvisionRisk** [dreamRs/shinyvs#9] [Q&A]
Special characters seems to be a challenge shinyvs::virtualSelectInput (at least for me). For example '&' is translated to html 'amp;'. Is there a way to suppress this behavior?

Here is a working example:
library(shiny)
library(shinyvs)

ui <- fluidPage(
  tags$h2("Virtual Select"),
  
  shinyvs::virtualSelectInput(
    inputId = "single",
    label = "Single select:",
    choices = c("test 1 & something", "test 2 & something else"),
    search = TRUE
  ),
  verbatimTextOutput("res_single"),
)

server <- function(input, output, session) {
  output$res_single <- renderPrint(input$single)
}

if (interactive())
  shinyApp(ui, server) 

Best wishes
Jonas

**Comment by pvictor** [comment]
It's in fact shiny which escapes the special characters, you have to use `html = TRUE` to make it work properly.

Victor
